README.md

```diff
@@ -25,25 +25,46 @@ Installation
 ------------
 IRBEM requires a [Fortran compiler](https://fortran-lang.org/learn/os_setup/install_gfortran), and can be installed on modern Windows, Linux, and Mac computers.
 
-- The quick build procedure on Linux with gfortran:
+## Linux
+The quick build procedure on Linux with gfortran:
 ```bash
     git clone https://github.com/PRBEM/IRBEM.git
     cd IRBEM
     make OS=linux64 ENV=gfortran64 all
     make OS=linux64 ENV=gfortran64 install
 ```
 
-- The quick build procedure on Mac OSX with gfortran:
+## Mac OSX
+The quick build procedure on Mac OSX with gfortran:
 ```bash
     git clone https://github.com/PRBEM/IRBEM.git
     cd IRBEM
     make OS=osx64 ENV=gfortran64 all
     make OS=osx64 ENV=gfortran64 install
 ```
 
-- The quick build procedure for Windows will be created soon. In the meanwhile, one workaround is to use the Windows Subsystem for Linux (WSL).
+## Windows
+The build procedure for 64-bit Windows with gfortran is involved, mainly due to the Fortran compiler. 
 
-See the `README.install` file for more details.
+1. Download and install [MSYS2](https://www.msys2.org/) and follow the [Fortran installation steps](https://www.msys2.org/#:~:text=and%20what%20for.-,Installation,-Download%20the%20installer) (summarized below). 
+   1. Run MSYS2 MSYS terminal from the start menu and update the MSYS2 packages via ```pacman -Syu``` (pacman is its package manager). Close the terminal after packages are updated. 
+   2. Run MSYS2 MSYS from the start menu again and update the rest of the packages using ```pacman -Syu```.
+   3. Install the Fortran compiler and other dependencies ([base-devel](https://packages.msys2.org/group/base-devel) and [mingw-w64-x86_64-toolchain](https://packages.msys2.org/group/mingw-w64-x86_64-toolchain)) in the MSYS terminal using ```pacman -S --needed base-devel mingw-w64-x86_64-toolchain```.
+   4. If you don't have git installed, install it using [msys2-git](https://packages.msys2.org/base/git) ```pacman -S git```.
```

Comment:
Git is actually required for IRBEM library installation (linking). Makefile uses git command to create a file with a version. You will not be able to build the library if just download it and you don't have git.

Reply:
I'm going to mark this as resolved, but the vcs never used to be required for build. There was a fallback such that a user could build from a source distribution. Requiring git for the build is a choice I strongly disagree with, but it's a separate issue so if it's _currently_ required then this is okay.

Reply:
@drsteve I also feel that it is uncessary, but removing the git commands only from the makefile will break at least one Fortran function, as well as the IDL and Matlab wrappers.

Reply:
So I just checked and the fallback is still in `Makefile`. It checks for the `.git` directory, and if that's present it assumes you have git (which is fair, I suppose)... if you download the source it won't find the `.git` directory, won't try to run any commands using git, and then will tag the release as `UNKNOWN`. I would hope the IDL/Matlab wrappers still work when built from a non-vcs source bundle.

This can probably be handled if/when actual versioned releases of the library start happening again. For that we'd expect folks to download a source distribution, but for each versioned release it should give the version rather than `UNKNOWN`. For now it's probably fine to just put the issue to one side.